recipes/menpo3d/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "menpo3d" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b369c2b30cba53cba758826bcb75960104ca4d1449f8262eb240f4861c22e58d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py<35]
+
+requirements:
+  build:
+     - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - cython
+    - numpy
```

Comment:
Are you linking against numpy? You may need numpy in the run section per https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Yes apologies - added the `pin_compatible`